web/src/components/merchant/sign-in/SignInForm.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import FormRow from 'components/common/FormRow';
+import useSignInForm from 'components/merchant/sign-in/hooks/useSignInForm';
+import Button from 'muicss/lib/react/button';
+import Form from 'muicss/lib/react/form';
+import styled from 'styled-components';
+
+const StyledForm = styled(Form)`
```

Comment:
Can you re-use the styles between sign-in and sign-up forms? They look so similar; some styles look identical (e.g. StyledForm).

Reply:
Okay! Will refactor similar components out to reduce duplication.